packages/react-instantsearch-core/src/connectors/connectQueryRules.ts

```diff
@@ -108,24 +106,17 @@ function getRuleContextsFromTrackedFilters({
   return ruleContexts;
 }
 
-export default createConnector({
+export default createConnector<QueryRulesProps>({
   displayName: 'AlgoliaQueryRules',
 
   defaultProps: {
     transformItems: items => items,
     transformRuleContexts: ruleContexts => ruleContexts,
     trackedFilters: {},
```

Comment:
I've just noticed that the type of the props is not correct, only `trackedFilters` is required.

Reply:
Actually no props are required at the connector level.

Reply:
`transformItems`, `transformRuleContexts` are required at the connector level? It would surprise me, otherwise it means that we can't use it like this (see below) inside the stories.

https://github.com/algolia/react-instantsearch/blob/c95523d792177ad5f961b3e5debbc6f493b14003/stories/QueryRuleContext.stories.tsx#L79-L83

Reply:
Nope, my message was probably badly written. I meant all props are optional in the connector.

Reply:
Oh ok, then all the types are incorrect. All of them should be optional.

Reply:
I'll take care of it.